pkg/runtime/runtime.go

```diff
@@ -447,6 +447,13 @@ func (a *DaprRuntime) setupTracing(ctx context.Context, hostAddress string, tpSt
 	return nil
 }
 
+func getOtelServiceName(fallback string) string {
+	if value, ok := os.LookupEnv("OTEL_SERVICE_NAME"); ok {
+		return value
+	}
```

Comment:
If the variable is set but empty, we should ignore it.
```suggestion
	if value := os.Getenv("OTEL_SERVICE_NAME"); value != "" {
		return value
	}
```